packages/vxrn/src/exports/prebuild.ts

```diff
@@ -111,18 +111,9 @@ export const prebuild = async ({
         'react,react-native,expo',
       ])
 
-      try {
-        const packageJsonPath = path.join(root, 'package.json')
-        let packageJsonContents = await FSExtra.readFile(packageJsonPath, 'utf8')
-
-        packageJsonContents = packageJsonContents.replace(/expo run:ios/g, 'one run:ios')
-        packageJsonContents = packageJsonContents.replace(/expo run:android/g, 'one run:android')
-
-        await FSExtra.writeFile(packageJsonPath, packageJsonContents, 'utf8')
-      } catch (error) {
-        throw new Error('Error updating package.json ' + error)
-      }
-
+      console.warn('expo unfortunatly added some scripts in package.json')
+      console.log('please replace "expo run:ios" with "one run:ios"')
+      console.log('please replace "expo run:android" with "one run:android"')
```

Comment:
Is there a reason for removing this automatic replacement? I think we can still try to do this automatically, and if failed, show a warning with manual instructions. 

As there are several duplicated issues opened before by people not noticing this, and the expo commands will start Metro automatically, making weird errors while trying to run RN.